spec/author_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'date'
+require_relative 'spec_helper'
+
+describe Author do
```

Comment:
From the project requirement, we should test all implemented methods. Well done you have tested some of the `author` methods but not all, please kindly test the `add_author` and the `list_authors` methods in your `author_module.rb`